dbos/_dbos.py

```diff
@@ -113,6 +115,13 @@
 from ._logger import add_otlp_to_all_loggers, config_logger, dbos_logger, init_logger
 from ._workflow_commands import get_workflow, list_workflow_steps
 
+try:
```

Comment:
Can we just populate the globalParams?

Reply:
I don't think we'd reuse it anywhere else. The CLI path and the `DBOS()` path don't intersect.

Reply:
Let's put it in GlobalParams, so it's only in one place instead of two, and so if more places need it they have somewhere to pull from.